__tests__/utils/desktop.js

```diff
@@ -0,0 +1,44 @@
+import {createInstance} from 'osjs';
+import {resourceResolver} from '../../src/utils/desktop';
+
+describe('#resourceResolver icon themes', () => {
```

Comment:
I think this can be condensed a bit. Instead of having all of these cases, maybe we can just compare arrays ?

```js
test('Should resolve correct paths', () => {
  const input = [
    'test.png',
    // and so on...
  ]

  const output = input.map(resolver.icon)

  expect(output).toEqual([
    '/icons/GnomeIcons/icons/test.png',
   // and so on...
  ])
})

```

Not a request, but something to look at it you want to.